src/sub-blocks/PriceCard/PriceCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Check from '@gravity-ui/icons/Check';
+
+import {Button, CardBase, ContentList, Link as LinkBlock} from '../../components';
+import {PriceCardProps} from '../../models';
+import {block} from '../../utils';
+
+import './PriceCard.scss';
+
+const b = block('price-card');
+
+const PriceCard = (props: PriceCardProps) => {
+    const {
+        border,
+        title,
+        price,
+        pricePeriod,
+        priceDetails,
+        theme,
+        description,
+        list,
+        button,
+        link,
+        backgroundColor,
+    } = props;
+    return (
+        <CardBase className={b({theme})} border={border}>
+            <CardBase.Content>
+                <div className={b('bg')} style={{backgroundColor}} />
+                <div className={b('content', {theme})}>
+                    <div className={b('info')}>
+                        <div className={b('title')}>{title}</div>
+                        <div className={b('price')}>
+                            <div>
+                                <span className={b('price-value')}>{price}</span>
+                                {pricePeriod && (
+                                    <span className={b('price-period')}>/ {pricePeriod}</span>
+                                )}
+                            </div>
+                            {priceDetails && (
+                                <div className={b('price-details')}>{priceDetails}</div>
+                            )}
+                        </div>
+                        <div>{description}</div>
+                        {list?.length ? (
+                            <div className={b('list')}>
+                                <ContentList
+                                    list={list.map((text) => ({
+                                        icon: Check,
+                                        text,
+                                    }))}
+                                    size="s"
+                                />
+                            </div>
+                        ) : null}
+                    </div>
+                    {button && <Button className={b('button')} {...button} />}
+                    {link && <LinkBlock className={b('link')} textSize="m" {...link} />}
```

Comment:
if there is an empty object, will it work correctly?

Reply:
No it won't. @yuberdysheva Should i fix this in src/sub-blocks/BackgroundCard/BackgroundCard.tsx too?
What would be the expected behaviour? (Don't show these elements until they get all required props, do they)

Reply:
> if there is an empty object, will it work correctly?

I've just realized that case with empty object is not possible. `interface LinkProps` has required `url` prop so it can't be set from the code. The other scenarios would be handled by validator.
There's just one situation with setting empty object on the storybook interface manually but it can be done for any other field either.